README.md

```diff
@@ -0,0 +1,400 @@
+# MOA Command Line Tool
+
+This project contains the `moactl` command line tool that simplifies the use of Amazon Red Hat OpenShift, also known as _MOA_.
+
+# Quickstart guide
+
+This guide walks through setting up your first Amazon Red Hat OpenShift cluster using `moactl`.
+
+This includes the following steps:
+
+* Installation prerequisites
+* Initializing your AWS account
+* Creating your cluster
+* Accessing your cluster
+* Installing an addon to your cluster
+* Creating admin users to your cluster
+* Cleaning up
+
+By the end of this guide you will have a Amazon Red Hat OpenShift cluster running in your AWS account.
+
+## Installation prerequisites
+
+Complete the following prerequisites before creating your MOA cluster.
+
+### Select an AWS account to use
+
+Unless your just testing out MOA, we recommend using a dedicated AWS account to run any production clusters. If you are utilizing AWS Organizations, you can use an AWS account within your organization or [create a new one](https://docs.aws.amazon.com/organizations/latest/userguide/orgs_manage_accounts_create.html#orgs_manage_accounts_create-new).
+
+Next, you must not have a Service Control Policy applied to the AWS account you plan to use. As part of the cluster creation process, moactl will perform the following actions:
+
+- Create an osdCcsAdmin IAM user:
+  - This user will have Programmatic access enabled.
+  - This user will have the AdministratorAccess policy attached to it.
+  - This user will leverage the IAM credentials you provide when configuring the AWS cli in the next section.
```

Comment:
@lamek it is OK to have an SCP applied to the AWS account you wish to use. It must be either allow all or have the following policy applied or equivalent permissions https://www.openshift.com/dedicated/ccs

Reply:
> > @jeremyeder @vkareh @jharrington22 PTAL. 2 questions specifically I could use help on.
> 
> RE Np. 2 - We don't have a publicly listed set of required quota. We could add a feature for `moactl` to print what it is validating? Thoughts?

I think this would be very helpful. Depending on the amount of work, it might be worth waiting to see how many people stumble on this.